frontend/lib/api/ApiClient.test.ts

```diff
@@ -90,14 +90,4 @@ describe("ApiClient", () => {
 
     expect(console.error).toHaveBeenCalledWith("Unexpected data from server:", expect.any(String));
   });
-
-  test("Unexpected status code throws an error", async () => {
```

Comment:
This test originally was testing a 201 response to a GET request. Later the response was changed to a 301, making it a duplicate of the test "318 response throws an error".

I don't think we have any code for the GET that makes it behave differently if it gets a 201 instead of a 200, so resolving this thread, since this test should indeed be deleted.